### ISSUE TITLE
Bump up to NSP v2.0.0

### ISSUE THREAD

**Issue by valeriansaliou** [FGRibreau/check-build#51]
Hey there,

Looks like current check-build NSP version is broken (v1.1.0), I constantly get:

```
/usr/local/lib/node_modules/nsp/node_modules/text-table/index.js:59
    return m ? m.index + 1 : c.length;
                              ^

TypeError: Cannot read property 'length' of undefined
    at dotindex (/usr/local/lib/node_modules/nsp/node_modules/text-table/index.js:59:31)
    at /usr/local/lib/node_modules/nsp/node_modules/text-table/index.js:11:21
    at Array.forEach (native)
    at forEach (/usr/local/lib/node_modules/nsp/node_modules/text-table/index.js:73:31)
    at /usr/local/lib/node_modules/nsp/node_modules/text-table/index.js:10:9
    at Array.reduce (native)
    at reduce (/usr/local/lib/node_modules/nsp/node_modules/text-table/index.js:63:30)
    at module.exports (/usr/local/lib/node_modules/nsp/node_modules/text-table/index.js:9:20)
    at prettyOutput (/usr/local/lib/node_modules/nsp/lib/prettyOutput.js:32:17)
    at /usr/local/lib/node_modules/nsp/commands/auditPackage/index.js:35:9
```

-- Which breaks my local tests + CI + ...production deploys.

Would be great if you could bump it up to the newly-available, bleeding edge NSP v2.0.0.

:beers:

**Comment by mguihal** [comment]
:+1:

**Comment by FGRibreau** [comment]
fixed in #52, v2.3.0 :)
